config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-release-1.25.yaml

```diff
@@ -104,7 +137,7 @@ presubmits:
         - runner.sh
         env:
         - name: KUBERNETES_VERSION
-          value: "v1.24.3"
+          value: "v1.24.16"
```

Comment:
Not yet available at https://hub.docker.com/r/kindest/node/tags?page=1&name=1.24